.changeset/wet-boxes-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@emotion/babel-plugin-jsx-pragmatic': minor
+---
+
+Convert to TypeScript. No breaking changes
```

Comment:
```suggestion
Source code has been migrated to TypeScript. From now on type declarations will be emitted based on that, instead of being hand-written.
```